frontend/packages/pipelines-plugin/src/components/pipelines/modals/triggers/submit-utils.ts

```diff
@@ -71,47 +68,27 @@ export const submitTrigger = async (
     pipelineRun,
     triggerTemplateParams,
   );
+  const eventListener: EventListenerKind = await createEventListener(
```

Comment:
I know this predates this PR, but let's not keep this an async method. Do the fetching of the operator version out here and pass it into `createEventListener` instead. Just so the method is more pure with its usage.